### PR TITLE
Prepare Release v1.5.0

### PR DESCRIPTION
**Pull request by everett980**
## Short description of the changes
- Update `CHANGELOG.md` and set `packages/jaeger-ui/package.json#version` to v1.5.0.
- Update `scripts/get-changelog.js` to:
-- not crash on bot-generated PRs
-- and only output PRs not present in `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
.../github.com/jaegertracing/jaeger-ui/pull/405); and [@everett980](https://github.com/everett980) in [#468](https://github.com/jaegertracing/jaeger-ui/pull/468), [#466](https://github.com/jaegertracing/jaeger-ui/pull/466), [#458](https://github.com/jaegertracing/jaeger-ui/pull/458), [#463](https://github.com/jaegertracing/jaeger-ui/pull/463), [#457](https://github.com/jaegertracing/jaeger-ui/pull/457), [#453](https://github.com/jaegertracing/jaeger-ui/pull/453), [#452](https://github.com/jaegertracing/jaeger-ui/pull/452), [#434](https://github.com/jaegertracing/jaeger-ui/pull/434), [#433](https://github.com/jaegertracing/jaeger-ui/pull/433), [#432](https://github.com/jaegertracing/jaeger-ui/pull/432), [#429](https://github.com/jaegertracing/jaeger-ui/pull/429), [#427](https://github.com/jaegertracing/jaeger-ui/pull/427), [#424](https://github.com/jaegertracing/jaeger-ui/pull/424), [#421](https://github.com/jaegertracing/jaeger-ui/pull/421), [#412](https://github.com/jaegertracing/jaeger-ui/pull/412), [#398](https://github.com/jaegertracing/jaeger-ui/pull/398), [#386](https://github.com/jaegertracing/jaeger-ui/pull/386), and [#378](https://github.com/jaegertracing/jaeger-ui/pull/378))
```
Comment (yurishkuro) [inline CHANGELOG.md]: could we just book an issue and reference it here, and the linkage to PRs can be from the issue itself. If I am a user reading this CL, I probably don't care about every commit that went into implementing the feature.

Comment (yurishkuro) [reply CHANGELOG.md]: also, the title says "improve DDG", could probably use a bit more description if there's some significant improvement

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+- **Deep Dependency Graph:** Derive DDG from search results ([@rubenvp8510](https://github.com/rubenvp8510) in [#445](https://github.com/jaegertracing/jaeger-ui/pull/445))
+
+- **Configuration:** Add script tags from config to body ([@everett980](https://github.com/everett980) in [#455](https://github.com/jaegertracing/jaeger-ui/pull/455))
```
Comment (yurishkuro) [inline CHANGELOG.md]: can we rephrase this? without looking into the ticket I don't know what this means

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+
+- **Plexus:** Sequester zoom concerns to ZoomManager ([@tiffon](https://github.com/tiffon) in [#409](https://github.com/jaegertracing/jaeger-ui/pull/409))
+
+- **Plexus:** Support multiple layers ([@tiffon](https://github.com/tiffon) in [#426](https://github.com/jaegertracing/jaeger-ui/pull/426), [#428](https://github.com/jaegertracing/jaeger-ui/pull/428), [#422](https://github.com/jaegertracing/jaeger-ui/pull/422), [#420](https://github.com/jaegertracing/jaeger-ui/pull/420), [#419](https://github.com/jaegertracing/jaeger-ui/pull/419), [#416](https://github.com/jaegertracing/jaeger-ui/pull/416), and [#414](https://github.com/jaegertracing/jaeger-ui/pull/414))
```
Comment (yurishkuro) [inline CHANGELOG.md]: similar, could be just a single issue

File: CHANGELOG.md (hunk 1 above)
Comment (everett980) [reply CHANGELOG.md]: yeah I didn't love this approach, but felt it was better than listing every PR and didn't want to lose the tie between the `CHANGELOG.md` and the PRs if every wanted.

an intermediary issue is even better.

Comment (yurishkuro) [reply CHANGELOG.md]: you can simply move this list of diffs to an issue

**Pull request merged**